packages/beacon-node/src/chain/chain.ts

```diff
@@ -505,6 +506,36 @@ export class BeaconChain implements IBeaconChain {
     }
   }
 
+  prunePastInvalidSszObjects(): void {
```

Comment:
I see. I put `prunePastInvalidSszObjects` here because there is the method `persistInvalidSszObject`, which writes to disk. The rationale being both write and delete being in the same module.